src/morphology/english/regexAdjective.js

```diff
@@ -0,0 +1,149 @@
+const noComparativeOrSuperlative = new RegExp( /(.)(ing|s|c|ful|ish|al|ive|like|ed|ble)$/, "i" );
+
+const comparative = [
+	[ /([^aeiou])y$/i, "$1ier" ],
+	[ /(..)e$/i, "$1er" ],
+	[ /([plgmrv]id)$/i, "$1er" ],
+	[ /([^aeiouy])([aeiouy])([bdfglmnpt])$/i, "$1$2$3$3er" ],
```

Comment:
Here and in `superlative`: There is a rule for adjectives that end in `b` (like `drab`) to make comparative and superlative forms. However, there is no rule that bring their comparatives and superlatives to base (`drabber` >  `drab`, `drabbest` > `drab`). Please add the rules to `comparativeToBase` and `superlativeToBase`.